packages/storybook/stories/ControlBar/Buttons/ControlBarButton/__snapshots__/ControlBarButton.stories.storyshot

```diff
@@ -38,6 +38,7 @@ exports[`storybook snapshot tests Storyshots UI Components/ControlBar/Buttons/De
           <span
             aria-hidden={true}
             className="root-span"
+            primaryFill="currentColor"
```

Comment:
Not sure why these are being update when I run `rushx test -u` in packages/storybook. I went back to main and ran `rushx test -u` and this also happens.

Reply:
I just pushed a fix that removed these, you'll run to run `rush update` to ensure storybook tests are being run with the latest `@fluent/react-icons` package to ensure these don't show up

Reply:
Ah ok. Will do that. Thanks!